addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/CookieSameSiteScanRuleUnitTest.java

```diff
@@ -359,25 +294,32 @@ void shouldAlertWhenCookieNotOnIgnoreList() throws HttpMalformedHeaderException
                 .getConfig()
                 .setProperty(RuleConfigParam.RULE_COOKIE_IGNORE_LIST, "aaaa,bbb,ccc");
 
+        HttpMessage msg = createMessage("");
+        msg.getResponseHeader()
+                .setHeader(HttpResponseHeader.SET_COOKIE, "test=123; Path=/; HttpOnly");
+        // When
+        scanHttpResponseReceive(msg);
+        // Then
+        assertThat(alertsRaised.size(), equalTo(1));
+        assertThat(alertsRaised.get(0).getParam(), equalTo("test"));
+        assertThat(alertsRaised.get(0).getEvidence(), equalTo("Set-Cookie: test"));
+    }
+
+    private HttpMessage createMessage(String sameSite) throws HttpMalformedHeaderException {
```

Comment:
- `static`
 - It could be created the method `createMessage()` to avoid passing an empty string in the actual tests, it could also check the string, if empty don't add the Set-Cookie header.